### PR TITLE
🚚 Deprecate Hugo aliases

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

Hugo aliases use meta tags with `refresh` set to true.
For SEO, it's been suggested we should return 301 codes instead.

## What's changed

Changes aliases into redirects and added guidance on why.

### CODE REVIEW

File: .platform/routes.yaml [hunk 1]
```diff
@@ -132,6 +132,28 @@ https://{default}/:
...
+      "/frameworks/deploy-button.html":  { "to": "/development/deploy-button.html", "code": 301 }
+      "/development/public-ips.html":  { "to": "/development/regions.html", "code": 301 }
+      "/golive/steps/cloudflare.html":  { "to": "/domains/cdn/cloudflare.html", "code": 301 }
+      "/golive/steps/fastly.html":  { "to": "/domains/cdn/fastly.html", "code": 301 }
+      "/domains/quick-start.html":  { "to": "/domains/steps.html", "code": 301 }
+      "/administration/web/environments.html":  { "to": "/environments.html", "code": 301 }
+      "/guides/general/default-branch.html":  { "to": "/environments/default-environment.html", "code": 301 }
+      "/development/logs.html":  { "to": "/increase-observability/access-logs.html", "code": 301 }
+      "/languages/nodejs/nvm.html":  { "to": "/languages/nodejs/node-version.html", "code": 301 }
+      "/guides/general/composer-auth.htmll":  { "to": "/languages/php/composer-auth.html", "code": 301 }
```
Comment (trolologuy) [inline .platform/routes.yaml:149]: :lady_beetle:
```suggestion
      "/guides/general/composer-auth.html":  { "to": "/languages/php/composer-auth.html", "code": 301 }
```

**Review by trolologuy — CHANGES_REQUESTED** [review]
Was there a specific reason to remove the multiline (`|`) descriptions?
Other than that, the redirections work (with a few typos).

**Comment by CollierCZ** [comment]
I feel like multiline markers are confusing for things that shouldn't be longer than one line :shrug:

**Review by trolologuy — DISMISSED** [review]
✅ The links work 😎

**Review by trolologuy — APPROVED** [review]
✅

**Pull request merged**